www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/Api/ServiceController.php

```diff
@@ -57,6 +57,27 @@ public function stopAction()
      * @throws \Exception when configd action fails
      * @throws \ReflectionException when model can't be instantiated
      */
+    public function reconfigureAction()
```

Comment:
I don't think you're listening. The parent class **provides** this.

Reply:
yup, found that out shortly afterwards. Doesn't work though (see issue #2581)

Reply:
If setup.php is called, that should be fine, but I'm not in a position to test on a cluster right now.